config/deploy.rb

```diff
@@ -51,4 +51,4 @@
 
 after "deploy:update", "deploy:cleanup"
 after "deploy:symlink", "deploy:link_folders"
-after "deploy:artisan_migrate", "deploy:artisan_cache_clear"
+after "deploy:link_folders", "deploy:artisan_migrate"
```

Comment:
I think we added that in the voting app, to fix something, but thinking about it here, would just mean we would wipe user info on every deploy, so we probably don't want that.